client/my-sites/comments/comment/comment-header.jsx

```diff
@@ -0,0 +1,49 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import Gridicon from 'gridicons';
+import noop from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import CommentAuthor from 'my-sites/comments/comment/comment-author';
+import CommentAuthorMoreInfo from 'my-sites/comments/comment/comment-author-more-info';
+import FormCheckbox from 'components/forms/form-checkbox';
+
+export const CommentHeader = ( {
+	commentId,
+	isBulkMode,
+	isEditMode,
+	isExpanded,
+	isSelected,
+	toggleExpanded,
+} ) => (
+	<div className="comment__header" onClick={ ! isExpanded && ! isBulkMode ? toggleExpanded : noop }>
```

Comment:
This suppresses one existing feature that I really liked. I can click on anywhere in the header to expand a comment, and then if I want to revert that quickly (e.g. expanded by mistake) I can click in the same place to do that. That's not possible anymore, since I have to move my cursor over the chevron on the right.

Judging by the border I'm assuming that that's by design. However, I'd still like to point this out so we can reconsider this decision (on the other hand I really like the fact that there is a border between `User info` and chevron when expanded). 

/cc @megsfulton

Reply:
To be fair, by design we're never intended to click on the entire header anymore (see p3fqKv-5W1-p2 prototypes).

I'll remove this `onClick`; let only the toggle button handle the expand; convert all the `CommentAuthor` spans with links in both expanded and collapsed views (as intended by the prototype); double check the broken ternary you pointed out in the next comment.